### PR TITLE
Add delete command

### PR DESCRIPTION
**Pull request by TeeRenJing**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/PrescriptionListParser.java [hunk 1]
```diff
@@ -1,19 +1,15 @@
 package seedu.address.logic.parser;
 
-import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.*;
```
Comment (shyanyong) [inline src/main/java/seedu/address/logic/parser/PrescriptionListParser.java:4]: import *

**Review by shyanyong — APPROVED** [review]
LGTM, other than the import statement and the checkstyle.

File: src/main/java/seedu/address/logic/parser/PrescriptionListParser.java [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
 
-import seedu.address.commons.core.LogsCenter;
-import seedu.address.logic.commands.AddPrescriptionCommand;
-import seedu.address.logic.commands.CommandPrescription;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListPrescriptionCommand;
-import seedu.address.logic.commands.ListTodayPrescriptionCommand;
-import seedu.address.logic.parser.exceptions.ParseException;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
```
Comment (RoeReRe) [inline src/main/java/seedu/address/logic/parser/PrescriptionListParser.java:5]: Actually I think theres a `MessagesPrescription` class. I think we omitted this in many previous PRs also T_T.

But nvm for now, can fix once we nuke the AB3 files.

**Review by RoeReRe — APPROVED** [review]
LGTM. But prepare to write test code for this before v1.2 deliverable (some existing AB3 ones are commented out u can check em out).

**Pull request merged**